README.md

```diff
@@ -62,13 +64,23 @@ If you wish you can run this command under a virtual environment.
 ## Running
 
 After installing the requirements, you can run the tests with the following
-commands::
+commands:
 
+```bash
   $ export PYTHONPATH=.:$PYTHONPATH
   $ avocado run --test-runner='nrunner' ./tests/domain/*.py
+```
 
-.. note:: The Next Runner (nrunner) will be the default runner soon in Avocado.
+Please note that the Next Runner (nrunner) will be the default runner soon in
+Avocado. So `--test-runner='nrunner'` can be removed.
 
 ## Writing Tests
 
-.. note:: TODO
+You can write your tests here the same way you write for Avocado Framework.
```

Comment:
"for the Avocado Framework" or "with the Avocado Framework" depending on the meaning.